addons/account/tests/test_account_journal_dashboard.py

```diff
@@ -180,11 +180,11 @@ def test_gap_in_sequence_warning(self):
                 'price_unit': 2.27,
                 'tax_ids': [],
             })]
-        } for i in range(12)]).sorted('date')
+        } for i in range(14)]).sorted('date')
         gap_date = moves[6].date
 
-        moves.action_post()
-        self.assertFalse(journal._query_has_sequence_holes())  # no gap, no gap warning
+        moves[:12].action_post()  # Only post 12 moves and keep 2 draft moves
+        self.assertFalse(journal._query_has_sequence_holes())  # no gap, no gap warning, and draft moves shouldn't trigger the warning
```

Comment:
Use the public API instead of the private methods
```suggestion
        self.assertFalse(journal.has_sequence_holes)  # no gap, no gap warning, and draft moves shouldn't trigger the warning
```